src/test/java/com/microsoft/sqlserver/testframework/AbstractTest.java

```diff
@@ -138,6 +143,9 @@ public static void setup() throws Exception {
         applicationKey = getConfiguredProperty("applicationKey");
         tenantID = getConfiguredProperty("tenantID");
 
+        accessTokenClientId = getConfiguredProperty("accessTokenClientId");
+        accessTokenSecret = getConfiguredProperty("accessTokenSecret");
```

Comment:
Can we re-use the existing principal ID and secret we already have?

Reply:
Yes, my mistake. After checking, there already is a pair secret/clientId available.